### PR TITLE
Update fix docs

### PR DESCRIPTION
**Pull request by lucasmoura**
This PR is aims at:

* Updating the output examples of `pro fix` for the latest improvement in the Pro client
* Add a how-to-guide to help users on better visualizing the status of each CVE when the fix execute API endpoint

### CODE REVIEW

File: docs/howtoguides/how_to_better_visualize_fixing_multiple_cves.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# How better visualized result of fixing multiple CVEs
```
Comment (renanrodrigo) [inline docs/howtoguides/how_to_better_visualize_fixing_multiple_cves.md:1]:
```suggestion
# How to better visualize the result when fixing multiple CVEs
```

File: docs/howtoguides/how_to_better_visualize_fixing_multiple_cves.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+We can see that the command output will be something that follows this structure here:
+
+```
+CVE-2020-28196 (Kerberos vulnerability) - fixed
+CVE-2020-15180 (MariaDB vulnerabilities) - not-affected
+CVE-2017-9233 (Coin3D vulnerability) - fixed
+```
+
+Note that this output consists of three information details:
```
Comment (renanrodrigo) [inline docs/howtoguides/how_to_better_visualize_fixing_multiple_cves.md:37]:
```suggestion
Note that each entry in this output consists of three fields:
```

**Review by renanrodrigo — APPROVED** [review]
couple suggestions, lgtm overall

**Comment by lucasmoura** [comment]
@s-makin CI is failing here due to a spellcheck error. However, the error is on the word `visualize`. I can change that word, but I just want to double check if we are following the British spelling here, and no problem at all if we are using, as I just want to double check if that is expected

**Comment by s-makin** [comment]
> @s-makin CI is failing here due to a spellcheck error. However, the error is on the word `visualize`. I can change that word, but I just want to double check if we are following the British spelling here, and no problem at all if we are using, as I just want to double check if that is expected

Yes, we are using British English so it should be "visualise". Global replace should get rid of the errors I think.

**Comment by lucasmoura** [comment]
@s-makin fixed

File: docs/howtoguides.rst [hunk 3]
```diff
@@ -92,3 +92,11 @@ Create a ``pro`` Golden Image
    :maxdepth: 1
 
    Create a customised Cloud Ubuntu Pro image <howtoguides/create_pro_golden_image.md>
+
+Better visualise result when fixing multiple CVEs
```
Comment (s-makin) [inline docs/howtoguides.rst:96]:
```suggestion
Fixing multiple CVEs
```

File: docs/howtoguides.rst [hunk 4]
```diff
@@ -92,3 +92,11 @@ Create a ``pro`` Golden Image
...
+==================================================
+
+.. toctree::
+   :maxdepth: 1
+
+   Better visualise result when fixing multiple CVEs <howtoguides/how_to_better_visualise_fixing_multiple_cves.rst>
```
Comment (s-makin) [inline docs/howtoguides.rst:102]:
```suggestion
   Better visualise results when fixing multiple CVEs <howtoguides/how_to_better_visualise_fixing_multiple_cves.rst>
```

File: docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
+.. _how_to_better_visualise_fixing_multiple_cves:
+
+How to better visualise the result when fixing multiple CVEs
+*************************************************************
+
+To fix multiple CVEs, you need to use the `u.pro.security.fix.cve.execute.v1` endpoint,
+as the CLI `pro fix` command doesn't support multiple CVEs. However, as can be seen :ref:`here <cve-execute-api-v1>`
```
Comment (s-makin) [inline docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst:7]:
```suggestion
as the CLI `pro fix` command doesn't support multiple CVEs. However, as can be seen :ref:`in the endpoint documentation <cve-execute-api-v1>`
```

File: docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+this command will output a JSON blob containing the results of the fix operation for each CVE.
+
+This doesn't allow for a quick visualization of the fix status of each requested CVE.
```
Comment (s-makin) [inline docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst:10]:
```suggestion
This doesn't allow for a quick visualisation of the fix status of each requested CVE.
```

File: docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+To address that, we can use a `jq` filter. The `jq` command is mainly used to parse JSON data directly on
```
Comment (s-makin) [inline docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst:11]:
```suggestion
To address that, we can use a `jq` filter. The `jq` command is mainly used to parse JSON data directly in
```

File: docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+the terminal. To know more about it, please refer to this `page <https://manpages.ubuntu.com/manpages/xenial/man1/jq.1.html>`_
```
Comment (s-makin) [inline docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst:12]:
```suggestion
the terminal. To know more about it, please refer to `the jq manpage <https://manpages.ubuntu.com/manpages/xenial/man1/jq.1.html>`_
```

File: docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+Before proceeding, we need to guarantee that `jq` is installed in your machine. This can be achieved
+by running the following command:
+
+.. code-block:: bash
+
+    $ apt update & apt install jq -y
+
+Now that `jq` is installed, we can properly parse the JSON data delivered from the execute API.
+As an example, let's assume we want to fix those three CVEs here: **CVE-2020-28196, CVE-2020-15180**
```
Comment (s-makin) [inline docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst:22]:
```suggestion
As an example, let's assume we want to fix these three CVEs: **CVE-2020-28196, CVE-2020-15180**
```

File: docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+  and **affected-until-reboot**.
+
+If you want to change the output format, you can tweak the `jq` filter. For example, to only show
+the CVE title and status, you can change the `jq` filter to:
+
+.. code-block:: bash
+
+    jq -r '.data.attributes.cves_data.cves[] | "\(.title) - \(.status)"'
+
+Finally, if you want to have the same visualization when fixing USNs, just change the `jq` filter
```
Comment (s-makin) [inline docs/howtoguides/how_to_better_visualise_fixing_multiple_cves.rst:54]:
```suggestion
Finally, if you want to have the same visualisation when fixing USNs, just change the `jq` filter
```

**Review by s-makin — COMMENTED** [review]
Only a couple of small nits, I think these might be why the spellcheck is still failing (hopefully should be gtg once fixed)

**Review by s-makin — APPROVED** [review]
Thanks @lucasmoura :) LGTM!

**Comment by s-makin** [comment]
> Thanks @lucasmoura :) LGTM!

Not going to merge - the spellchecker is complaining about jq and manpage, if you wouldn't mind adding those to the wordlist so that the checks can pass, you can feel free to merge it after

**Review by s-makin — APPROVED** [review]
LGTM :) thanks for all your work on this

**Pull request merged**